app/models/flood_risk_engine/organisation.rb

```diff
@@ -12,5 +12,11 @@ class Organisation < ActiveRecord::Base
       other: 5,
       unknown: 6
     }
+
+    has_one :primary_address, -> { where(address_type: Address.address_types["primary"]) },
+            class_name: "FloodRiskEngine::Address",
+            as: :addressable,
+            dependent: :restrict_with_exception
+
   end
 end
```

Comment:
from the RCDP model, which inserts a Contact class between a Party/Organisation and data such as postal_address, email etc sits in Contact - as per the WEX models